provisioning/roles/utils/celery/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Ensure supervisor is installed and up-to-date.
```

Comment:
Nope, shared templates as well. Not asking for it in that PR, just saying this is something to consider.